lib/rider.rb

```diff
@@ -0,0 +1,40 @@
+require 'csv'
+
+module RideShare
+
+  class Rider
+    attr_reader :rider_id, :name, :phone
+
+    def initialize(rider_info)
+      @rider_id = rider_info[:rider_id]
+      @name = rider_info[:name]
+      @phone = rider_info[:phone]
+    end
+
+    def self.all
+      @riders = []
+      CSV.foreach("support/riders.csv", {:headers => true}) do |line|
+        @riders << self.new({rider_id:line[0].to_i, name:line[1], vin:line[2]})
+      end
+      return @riders
+    end
+  end
+
+  def self.find(rider_id)
+    rider_lookup = RiderShare::Rider.all
+    rider_lookup.find { |rider| rider.rider_id == rider_id }
+    puts "invalid rider id" if rider_lookup == nil
```

Comment:
Probably don't need to `puts` here - returning `nil` is usually sufficient to let the caller know that nothing was found.